composer.json

```diff
@@ -17,11 +17,17 @@
         "dhii/php-cs-fixer-config": "dev-php-5.3",
         "phpunit/phpunit": "^4.8",
         "ptrofimov/xpmock": "^1.1",
-        "codeclimate/php-test-reporter": "<=0.3.2"
+        "codeclimate/php-test-reporter": "<=0.3.2",
+        "dhii/validation-interface": "^0.1"
```

Comment:
It's not optional, I'd say it's conditionally used.

Say I depend on this package in my application or library, and `$this->_validate()` throws a `ValidationFailedExceptionInterface`. That interface won't exist unless I also install developer dependencies, so PHP will throw a fatal error.